flowtools/convert_fcs_to_text/convertFCSToText.xml

```diff
@@ -1,13 +1,13 @@
-<tool id="convert_fcs_to_text" name="Convert FCS to Text" version="1.0">
+<tool id="convert_fcs_to_text" name="Convert FCS to Text" version="1.0+galaxy1">
   <description>with no transformation</description>
   <requirements>
-    <requirement type="package" version="2.0.2">ig-flowtools</requirement>
+    <requirement type="package" version="2.0.0">bioconductor-flowcore</requirement>
```

Comment:
```suggestion
    <requirement type="package" version="1.42.0">bioconductor-flowcore</requirement>
```
I would go for this version since it is what we are using in the other packages.

Reply:
We will need to re-check planemo tests after this change.